spec/support/shared_examples/cli/new.rb

```diff
@@ -44,7 +44,6 @@
       # lib/<project>.rb
       #
       expect("lib/#{project}.rb").to have_file_content <<-END
```

Comment:
@cllns Yeah, I'm gonna switch from HEREDOC to simpler string representation.

I was thinking to define a blank module like:

```ruby
module Bookshelf
end
```